.github/workflows/release-drafter.yml

```diff
@@ -16,11 +16,48 @@ on:
       - dev
 
 jobs:
+  build-docker-containers:
+    uses: ./.github/workflows/build-docker-images-for-testing.yml
+    secrets: inherit
+
+  oas-fetch:
+    needs: build-docker-containers
+    uses: ./.github/workflows/fetch-oas.yml
+    secrets: inherit
+
   update_release_draft:
+    needs: oas-fetch
     runs-on: ubuntu-latest
     steps:
-      - uses: release-drafter/release-drafter@v5.25.0
+      - name: Load OAS files from artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Create Release
+        id: create_release
+        uses: release-drafter/release-drafter@v5.25.0
         with:
           version: ${{github.event.inputs.version}}  
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload Release Asset - OpenAPI Specification - YAML
+        id: upload-release-asset-yaml
+        uses: sekwah41/upload-release-assets@v1
```

Comment:
Is there a reason to use this fork over `actions/upload-release-asset`? I know the official one is archived but I'm a little hesitant to use a random fork here..

Reply:
> Is there a reason to use this fork over `actions/upload-release-asset`? I know the official one is archived but I'm a little hesitant to use a random fork here..

As I remember, there is no other reason (only stopped maintenance).
If you prefer, I can run a local test with the original official one.

Reply:
Not sure if I'm doing sth wrong but I'm not able to make it work with `actions/upload-release-assets`
- https://github.com/kiblik/django-DefectDojo/actions/runs/7188225312/job/19577606201
- https://github.com/kiblik/django-DefectDojo/actions/runs/7188488164/job/19578368890

Any help is welcome